hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/PmemVolumeManager.java

```diff
@@ -283,52 +283,32 @@ void cleanup() {
    * Restore cache from the cached files in the configured pmem volumes.
    */
   public Map<ExtendedBlockId, MappableBlock> restoreCache(
-      String bpid, boolean isNative) throws IOException {
+      String bpid, MappableBlockLoader cacheLoader) throws IOException {
```

Comment:
judge cacheLoader is instance of PmemMappableBlockLoader. Then, convert it to PmemMapp...